src/libraries/Microsoft.Extensions.Logging.Configuration/src/LoggerFilterConfigureOptions.cs

```diff
@@ -52,8 +51,6 @@ private void LoadDefaultConfigValues(LoggerFilterOptions options)
                 }
             }
 
-            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
-                Justification = "IConfiguration.GetValue is safe when T is a bool.")]
             bool GetCaptureScopesValue(LoggerFilterOptions options) => _configuration.GetValue(nameof(options.CaptureScopes), options.CaptureScopes);
```

Comment:
This intermediate nested method can be removed now. It was only needed to put the UnconditionalSuppressMessage attribute on it.